packages/core/src/hydration/annotate.ts

```diff
@@ -251,7 +254,10 @@ export function annotateForHydration(appRef: ApplicationRef, doc: Document) {
   const serializedViews = serializedViewCollection.getAll();
   const transferState = injector.get(TransferState);
   transferState.set(NGH_DATA_KEY, serializedViews);
-  return eventTypesToReplay.size > 0 ? eventTypesToReplay : undefined;
+  return [
+    eventTypesToReplay.size > 0 ? eventTypesToReplay : undefined,
+    captureEventTypesToReplay.size > 0 ? captureEventTypesToReplay : undefined,
+  ];
```

Comment:
If we group both event sets into an object, we can return it from this function (with an extra wrapping object for future extension in case we need to add more fields in the future):
```suggestion
  return {eventTypesToReplay};
```

Reply:
I left it as eventTypesToReplay. When we do need to return more, we can do so. Just didn't wanna create an object.